{"name": "test_table_manufacturer", "status": "failed", "statusDetails": {"message": "AssertionError: assert (2,) == 2\n +  where (2,) = <bound method CMySQLCursor.fetchone of <mysql.connector.cursor_cext.CMySQLCursor object at 0x1180030d0>>()\n +    where <bound method CMySQLCursor.fetchone of <mysql.connector.cursor_cext.CMySQLCursor object at 0x1180030d0>> = <mysql.connector.cursor_cext.CMySQLCursor object at 0x1180030d0>.fetchone", "trace": "db_connection = <mysql.connector.connection_cext.CMySQLConnection object at 0x107f54eb0>\n\n    def test_table_manufacturer(db_connection):\n        with db_connection.cursor() as cursor:\n            with allure.step('INSERT в таблицу Manufacturer'):\n                cursor.execute('INSERT INTO manufacturer (name_manufacturer, country_id ) VALUES(\"Apple\", 1)')\n                db_connection.commit()\n                assert cursor.fetchone() is None\n    \n            with allure.step('Проверка результата INSERT'):\n                cursor.execute('SELECT * from manufacturer where name_manufacturer = %s', ('Apple',))\n                assert cursor.fetchone() is not None\n    \n            with allure.step('UPDATE в таблице Manufacturer'):\n                cursor.execute('UPDATE manufacturer SET country_id = %s WHERE name_manufacturer = %s', (2, 'Apple'))\n                db_connection.commit()\n                assert cursor.fetchone() is None\n    \n            with allure.step(\"Проверка результата UPDATE в таблице Manufacturer\"):\n                cursor.execute('SELECT country_id from manufacturer where name_manufacturer = %s', ('Apple',))\n>               assert cursor.fetchone() == 2\nE               assert (2,) == 2\nE                +  where (2,) = <bound method CMySQLCursor.fetchone of <mysql.connector.cursor_cext.CMySQLCursor object at 0x1180030d0>>()\nE                +    where <bound method CMySQLCursor.fetchone of <mysql.connector.cursor_cext.CMySQLCursor object at 0x1180030d0>> = <mysql.connector.cursor_cext.CMySQLCursor object at 0x1180030d0>.fetchone\n\ntest_db.py:46: AssertionError"}, "steps": [{"name": "INSERT в таблицу Manufacturer", "status": "passed", "start": 1698417249964, "stop": 1698417249966}, {"name": "Проверка результата INSERT", "status": "passed", "start": 1698417249966, "stop": 1698417249966}, {"name": "UPDATE в таблице Manufacturer", "status": "passed", "start": 1698417249966, "stop": 1698417249967}, {"name": "Проверка результата UPDATE в таблице Manufacturer", "status": "failed", "statusDetails": {"message": "AssertionError: assert (2,) == 2\n +  where (2,) = <bound method CMySQLCursor.fetchone of <mysql.connector.cursor_cext.CMySQLCursor object at 0x1180030d0>>()\n +    where <bound method CMySQLCursor.fetchone of <mysql.connector.cursor_cext.CMySQLCursor object at 0x1180030d0>> = <mysql.connector.cursor_cext.CMySQLCursor object at 0x1180030d0>.fetchone\n", "trace": "  File \"/Users/akopotov/Documents/College/Testing/checkpoint_6_dB_test/test_db.py\", line 46, in test_table_manufacturer\n    assert cursor.fetchone() == 2\n"}, "start": 1698417249967, "stop": 1698417249967}], "start": 1698417249964, "stop": 1698417249968, "uuid": "1ad85c01-839d-42f3-a92f-e91e5c35b5eb", "historyId": "7602d2145fb47dfe4ec3e603a1f4ee46", "testCaseId": "7602d2145fb47dfe4ec3e603a1f4ee46", "fullName": "test_db#test_table_manufacturer", "labels": [{"name": "suite", "value": "test_db"}, {"name": "host", "value": "alexMBP-14.local"}, {"name": "thread", "value": "45375-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_db"}]}